webview-ui/src/components/chat/ChatTextArea.tsx

```diff
@@ -23,6 +23,7 @@ import ApiOptions, { normalizeApiConfiguration } from "../settings/ApiOptions"
 import { MAX_IMAGES_PER_MESSAGE } from "./ChatView"
 import ContextMenu from "./ContextMenu"
 import { ChatSettings } from "../../../../src/shared/ChatSettings"
+import { a } from "vitest/dist/chunks/suite.d.FvehnV49.js"
```

Comment:
Please do not include compiled code in an import.  Also, this is importing "a" from vitest.  I feel this might have been an accidental import.